railties/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+*   Bump `NODE_LTS_VERSION` to 20.11.1
+
+    This value is used when conditionally generating a `.node-version` file when
+    generating new applications.
```

Comment:
The change makes sense, but the CHANGELOG should only mention user facing changes. 

```suggestion
*   Update the `.node-version` file conditionally generated for new applications to 20.11.1
```

(feel free to tweak this)